client/src/components/TODOList/AddNewTODO.js

```diff
@@ -117,39 +135,54 @@ const AddTODOItem = () => {
         Add
       </Button>
       <Dialog
-        size="lg"
+        size="md"
         open={openAddTODO}
         handler={handleOpen}
-        className="addTODODialog shadow-none"
+        className="shadow-none"
       >
-        <Card className="addTODOForm">
+        <Card className="addTODOForm max-w-xl">
           <h2>Add TODO Item</h2>
           <form onSubmit={handleSubmit}>
             <div className="inputField">
-              <label htmlFor="add-title">TODO:</label>
+              <label htmlFor="add-title">Title:</label>
               <Input
                 id="add-title"
                 name="title"
                 value={formData.title}
-                label="title"
+                label="Title"
                 onChange={handleInputChange}
               />
             </div>
-            <div className="inputField">
-              <label htmlFor="add-dueDate">Due Date:</label>
-              <DatePicker
```

Comment:
You could also consider the HTML datetime-local input type (or even the separated date and time types if you are worried about browser compatibility). It has better time granularity for one and its UI even seems cleaner: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local#try_it.

![image](https://github.com/awctw/study-dash/assets/56493743/a4123825-985c-4807-b26a-3e273ea96ac9)

Reply:
I have addressed this request in my latest commit